app/services/idv/data_url_image.rb

```diff
@@ -1,16 +1,32 @@
 module Idv
   class DataUrlImage
     def initialize(data_url)
-      @header, base64_data = data_url.split(',', 2)
-      @data = Base64.decode64(base64_data || '')
+      header, data = URI(data_url.chomp).opaque.to_s.split(',', 2)
```

Comment:
I like Ruby just a little bit more learning that there's a method named `chomp` 😄